oauth2_http/java/com/google/auth/oauth2/ImpersonatedCredentials.java

```diff
@@ -189,10 +251,31 @@ public String getAccount() {
     return this.targetPrincipal;
   }
 
+  @Override
+  public String getQuotaProjectId() {
+    return this.quotaProjectId;
+  }
+
+  public List<String> getDelegates() {
+    return delegates;
+  }
+
+  public List<String> getScopes() {
```

Comment:
This method doesn't seem used. Can it be removed?

Reply:
Same here. It is used to verify the scopes in tests. Please let me know if there are better ways to do it without defining this method.

Reply:
same

Reply:
Same here.